repos/spack_repo/builtin/packages/pumgen/package.py

```diff
@@ -14,7 +14,9 @@ class Pumgen(CMakePackage):
     homepage = "https://github.com/SeisSol/PUMGen/wiki/How-to-compile-PUMGen"
     git = "https://github.com/SeisSol/PUMGen.git"
     version("master", branch="master", submodules=True)
-
+    version(
```

Comment:
```suggestion

    version("master", branch="master", submodules=True)
    version(
```

We should maintain the space here between the package properties (URL, homepage, etc...) and the version directives.